### DISCUSSION TITLE
打包后多大呢？

### DISCUSSION THREAD

**Discussion by linonetwo** [YunYouJun/advjs#32] [General]
我在找一个轻量化的AVG玩法中间件，嵌入[我在设计的一个游戏化APP](https://wiki.onetwo.ren/#%E8%AE%BE%E8%AE%A1%E8%BD%BB%E9%87%8F%E7%89%88%E5%9F%BA%E9%87%91%E4%BC%9A%E7%BB%8F%E8%90%A5%E6%B8%B8%E6%88%8F%E7%94%A8%E4%BA%8E%E6%B8%B8%E6%88%8F%E5%8C%96)里，不过avg只是其中一个玩法，类似明日方舟里的过场剧情，我希望它不是太大，而且最好不依赖第三方框架，这样能小一点。

**Comment by linonetwo** [comment]
看了一下 webgal 用了 pixijs，实际上也不大 

![图片](https://github.com/YunYouJun/advjs/assets/3746270/16b6f336-056e-4451-9974-62db92cc38c9)

**Comment by YunYouJun** [comment]
> 我在找一个轻量化的AVG玩法中间件，嵌入[我在设计的一个游戏化APP](https://wiki.onetwo.ren/#%E8%AE%BE%E8%AE%A1%E8%BD%BB%E9%87%8F%E7%89%88%E5%9F%BA%E9%87%91%E4%BC%9A%E7%BB%8F%E8%90%A5%E6%B8%B8%E6%88%8F%E7%94%A8%E4%BA%8E%E6%B8%B8%E6%88%8F%E5%8C%96)里，不过avg只是其中一个玩法，类似明日方舟里的过场剧情，我希望它不是太大，而且最好不依赖第三方框架，这样能小一点。

之前的目的也是类似的用途。
依赖 Pixi.JS 的体积还好，依赖了可以做更多绘图上的交互还是值得的。

**Reply by linonetwo** [reply]
是的，我实际打包了一下，发现 WebGAL 去掉 React 和 Redux 之后也就 2M 以内。

抱歉这次没有采用 advjs，虽然一开始看用的脚本语言比 WebGAL 项目自己撮的脚本语言更加科学，但看到 advjs 的核心模块还一直在开发，而我周末就想写，所以先选用了 WebGAL。还是感谢你的讨论。
